py/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy==1.22.1
+opencv-python==4.5.5.62
+pkg_resources==0.0.0
```

Comment:
Når jeg åpnet prosjektet i Pycharm community edition, så fikk jeg feilmelding på pkg_resources==0.0.0 som jeg ikke klarte å løse. Jeg endte opp med å ignorere den. Koden ser ut til å fungere uten. Har fått bilde på skjermen i alle fall.

Reply:
Fjernet denne linjen, sikkert der pga https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command